django-stubs/db/models/query.pyi

```diff
@@ -140,20 +140,20 @@ _Row = TypeVar("_Row", covariant=True)
 class _ValuesQuerySet(Generic[_T, _Row], Collection[_Row], Reversible[_Row], QuerySet[_T], Sized): # type: ignore
     def __len__(self) -> int: ...
     def __contains__(self, x: object) -> bool: ...
-    def __iter__(self) -> Iterator[_Row]: ...  # type: ignore
-    @overload  # type: ignore
+    def __iter__(self) -> Iterator[_Row]: ...
+    @overload
     def __getitem__(self, i: int) -> _Row: ...
     @overload
     def __getitem__(self: _QS, s: slice) -> _QS: ... # type: ignore
```

Comment:
probably makes sense to leave a comment explaining why this is needed for the next person - but wanted to check with what the best way to phrase it is